assemble/bin/accumulo-cluster

```diff
@@ -157,16 +161,33 @@ function control_service() {
       # using the value of $host
       #
       if [[ $# -gt 3 ]]; then
-        ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd" "-a" "$host" "${@:4}"
+        EXTRA_ARGS="${*:4}"
```

Comment:
I'm not sure this `EXTRA_ARGS` is working the same as it did before. Here, it is flattened to a scalar and used as a single argument, whereas before, `"${@:4}"` was a sliced array and every extra argument would be considered its own argument. This will probably break things like using `-o key1=value1 -o key2=value2`, etc.

This is very different than the other place where `EXTRA_ARGS` is used in a very carefully quoted way when passing over SSH, so that each argument is treated as separate.